ui/app/models/ssh/ca-config.js

```diff
@@ -38,16 +38,15 @@ export default class SshCaConfig extends Model {
   @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
   @attr('string', { sensitive: true }) privateKey; // obfuscated, never returned by API
   @attr('string') publicKey;
```

Comment:
nothing changed here, just restructured so the pattern matches that in gcp and azure.